app/src/main/java/com/github/factotum_sdp/factotum/ui/roadbook/RoadBookFragment.kt

```diff
@@ -58,6 +62,14 @@ class RoadBookFragment : Fragment(), MenuProvider {
         rbRecyclerView.layoutManager = LinearLayoutManager(context)
         rbRecyclerView.adapter = adapter
 
+        locationTrackingHandler.setOnLocationUpdate {
+            val lat = it.latitude.toString().takeLast(3)
+            val long = it.longitude.toString().takeLast(3)
+            val cal = Calendar.getInstance()
+            println("Location from roadbookFrag: ($lat, $long)")
```

Comment:
It's not really annoying but I guess you don't need anymore the print statement ?